src/ripple/app/misc/impl/ValidatorKeys.cpp

```diff
@@ -22,12 +22,14 @@
 
 #include <ripple/app/misc/Manifest.h>
 #include <ripple/basics/Log.h>
+#include <ripple/basics/random.h>
 #include <ripple/core/Config.h>
 #include <ripple/core/ConfigSections.h>
 #include <beast/core/detail/base64.hpp>
 
 namespace ripple {
 ValidatorKeys::ValidatorKeys(Config const& config, beast::Journal j)
+    : cookie{rand_int<std::uint64_t>()}
```

Comment:
I figured since 0 meant "no cookie", there was little harm since if you get that unlucky, the worst that happens is you don't check that validator. But I'll fix, its really not much effort.